content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins.adoc

```diff
@@ -73,7 +73,7 @@ A reverse proxy must handle the HTTP response by either rewriting the response o
 When HTTP response handling is misconfigured, Jenkins may fail to show updated information on a page or it may ignore changes submitted through web pages.
 See the link:../reverse-proxy-configuration-troubleshooting/[troubleshooting] section if Jenkins is reporting that your reverse proxy setup is broken or pages are not behaving as expected.
 
-== Configuration Examples
+== Configuration Examples by Server Type
```

Comment:
We prefer to assign anchors (HTML `id`) to headings that are renamed so that URL's that reference the original heading will continue to open to that location in the page.
```suggestion
[[configuration-examples]]
== Configuration Examples by Server Type
```